datalad_container/adapters/docker.py

```diff
@@ -130,8 +134,10 @@ def cli_run(namespace):
               # Make it possible for the output files to be added to the
               # dataset without the user needing to manually adjust the
               # permissions.
-              "-u", "{}:{}".format(os.getuid(), os.getgid()),
               "--rm"]
+    if not on_windows:
+        prefix_extend(["-u", "{}:{}".format(os.getuid(), os.getgid())])
```

Comment:
s/_extend/.extend/

Reply:
argh, thanks for catching that! I hate notepad++. It so easy to lose/miss edits :/